websites/T/Tiermaker/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "tiermaker",
+    "author": {
+      "name": "Eason",
+      "id": "462456154266796032"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/FV8qLce.png",
+    "thumbnail": "https://i.imgur.com/5C3VsD2.png",
+    "version": "1.0.0",
+    "color": "#1b1c1c",
+    "tags": [
+      "anime"
+    ],

```

Comment:
This can go in 1 line

Reply:
![image](https://user-images.githubusercontent.com/71189551/145916754-2a555279-7db5-4127-ba4a-fde2729c6e4f.png)

Reply:
Yeah it can still go in one line

Reply:
oh it get it, yeah ill fix it.